### ISSUE TITLE
Accessibility Fix for Libcal

### ISSUE THREAD

**Issue by kevinreiss** [pulibrary/libapps#11]
The following custom css is causing a WCAG violation in libcal
```
.btn-primary:hover,
.btn-primary:active,
.btn-primary:focus {
background-color: #e87411;
border: 0;
}
```
The :focus state on buttons and many form elements at https://libcal.princeton.edu/appointments has very little contrast in Firefox.
WCAG Rule

1.4.11 AA: User interface components and states (icons, active indicators) have a contrast ratio of 3:1 against adjacent color(s). 

From the OIT key sites audit.
